### DISCUSSION TITLE
Update item context menu to show Move To and Copy To

### DISCUSSION THREAD

**Discussion by advice2020** [doublecmd/doublecmd#1783] [Suggestions]
The context menu provided by DC is very well done, very simple and clean limited amount of words used so not confusing.  I have seen other FM context menus that sometimes look very busy.
That being said, I was wondering if it would be a good idea to change
Move
Copy
in the context menu to
Move To
Copy To

Reason I say this is that the context menu contains 2 "copy" items
I rarely use context menu for these actions but I have on occassion used context menu to "copy" as in copy and paste and accidently hit the Copy (Copy To   F5) item in this context menu which performed a Copy To to my destination pane.

Not an important fix, again not something I use that often, but just something I noticed.
Not sure if any other users would be interested.
Thank You

**Comment by Skif-off** [comment]
This doesn't seem like a good idea because "Copy To" associate with a choosing a directory (like "Save As"), but there is no choice here.

P.S. As a last resort, you can copy "doublecmd.pot" to "doublecmd.en.po" and add the lines that are convenient for you (in `msgstr ""`, as "translation"). Although in this case you will have to sync files from time to time, but not very often. (I have been using my copy of the translation for a long time and I can't say that it is somehow burdensome.)

**Reply by alexx2000** [reply]
I agree that these menu items confuse users.

Maybe they should be removed at all.

**Comment by advice2020** [comment]
@Skif-off 
Ahh good point, totally forgot about this.
In the past when using different file managers have seen "copy to" and "move to" used as labels for these actions and 
others I have seen use "send to".

@alexx2000 
Decision is not up to me of course, but I can see what you are saying about maybe removing them as now they can be viewed/mistaken for three different actions.

@Anyone
Just opinions
I do not think these were up for debate but I think I would still keep the other two items in its context menu section, "Delete" and "Rename" unless feel otherwise.

Not important but just mentioning because there is talk of modifying this context menu.
I also noticed "Show File Properties"
I think this was done for the "F" Alt+F hotkey but maybe could consider changing this to "Properties"
I know "P" is taken by "Paste" but maybe could use something like "R" or "S"
If want to keep the theme of single word context menu items.
Have seen many FMs use just "Properites" at the bottom of context menu.
Again just minor opinions.

Thank you guys again for your responses
